docs/sources/mimir/migration-guide/migrating-from-cortex.md

```diff
@@ -108,10 +108,11 @@ It provides a simple migration by generating Mimir configuration from Cortex con
 
 [`mimirtool`]({{< relref "../operators-guide/tools/mimirtool.md" >}}) provides a command for converting Cortex configuration to Mimir configuration that you can use to update both flags and configuration files.
 
-### Download mimirtool
+### Install mimirtool
 
-Download the appropriate [release asset](https://github.com/grafana/mimir/releases/latest) for your operating system and architecture and make it executable.
-For Linux with the AMD64 architecture:
+To install Mimirtool, download the appropriate binary from the [latest release](https://github.com/grafana/mimir/releases/latest) for your operating system and architecture and make it executable.
+
+Alternatively, use a command line tool such as `curl` to download mimirtool. For example, for Linux with the AMD64 architecture:
```

Comment:
```suggestion
Alternatively, use a command line tool such as `curl` to download `mimirtool`. For example, for Linux with the AMD64 architecture, use the following command:
```
Use stand-alone phrases or sentences before code or code blocks, and in general where possible.